P2P.md

```diff
@@ -0,0 +1,20 @@
+# Prerequirements for building
+
+1. Install rust
+2. make fetch-thirdparty
+3. make setup-thirdparty
+
+# How to run silius with p2p enabled
+
+## Run silius with bootnodes
```

Comment:
I think we can structure this document as:
- Run Silius as bootnode
- Run Silius with bootnodes
- Run cluster of Silius bundler (here we can show example of starting multiple bundlers with correct ports, so you can just copy paste commands)